Fix: aria-expanded attribute added to navigation button (#96)

joe-allen-89 opened the pull request:
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
#96 

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* aria-expanded attribute added to navigation button (fixes #96)

kirsty-hames reviewed (commented):
Thanks for picking this up @joe-allen-89.

When `_location` is set to `notify` this works as expected. When the `_location` is set to `drawer` however the `aria-expanded` remains `true` when drawer is collapsed.

joe-allen-89 commented:
> Thanks for picking this up @joe-allen-89.
> 
> When `_location` is set to `notify` this works as expected. When the `_location` is set to `drawer` however the `aria-expanded` remains `true` when drawer is collapsed.

Apologies, stupidly missed off the event listener for drawer:closed. 

I've added that in and moved all event listeners to their own function to keep code clean.
